### ISSUE TITLE
Break up phases

### ISSUE THREAD

**Issue by aplarson** [Conanza/cuteorboot#1]
2 and 3 should be more granular

**Comment by Conanza** [comment]
Right. I'm still reworking the details; I'm thinking being able to upload/store photos is absolutely integral to this app, but having a profile to view/edit needs to come first in order to display those photos. Here's a general breakdown I have so far:

## Phase 2a: View/edit own profile (1-1.5 days)
First part of this, will be setting up the API endpoints to serve user data to BB.
* BB model: User, Picture, Hobby (Will need User to parse pictures and hobbies associations)
* BB collection: Users, Pictures, Hobbies

Then, create the views involved in creating the general layout of the logged-in state.
* Navbar: have the router install this on initialization in a 'sidebar'
* UserDashboard CompositeView (contains: UserNav 'header', UserDetail, UserLanding): this will go in a 'content' section

## Phase 2b: View/edit own profile (~1 day)
Create an EditProfile view, AddPhoto View(modal)

## Phase 3: Upload, Adding Pictures (~ 1 Day)
Implement file uploading and storage. Get file uploads to persist to the db and storage system. I want to be able to show images that users have uploaded.

Rails: I think it will need a Pictures model, PicturesController (new, create, destroy).
Backbone: an AddPhoto View (modal)
Gems/Libraries: AWS S3, cloudinary, filepicker, paperclip
